index.html

```diff
@@ -634,6 +634,29 @@
   </div>
   <!-- ends Check FFV1 Fixity  -->
 
+  <!-- Check interlacement  -->
+  <span data-toggle="modal" data-target="#check_interlacement"><button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="bottom" title="This identifies interlacement patterns in a video file">Check interlacement </button></span>
+  <div id="check_interlacement" class="modal fade" tabindex="-1" role="dialog">
+    <div class="modal-dialog modal-lg">
+      <div class="modal-content">
+        <div class="well">
+          <h3>Check video file interlacement patterns</h3>
+          <p><code>ffmpeg -i <i>input file</i> -vf idet -f null -</code></p>
+          <p></p>
+          <dl>
+            <dt>ffmpeg</dt><dd>starts the command</dd>
+            <dt>-i <i>input_file</i></dt><dd>path, name and extension of the input file</dd>
+            <dt>-vf idet</dt><dd>This calls the <a href="http://www.tate.org.uk/research/publications/sustaining-consistent-video-presentation">idet (detect video interlacing type) filter</a>.</dd>
```

Comment:
No, I agree, especially for beginners it is good to see consistency in a verbose manner.